src/main/scala/nl.knaw.dans.easy.deposit/StateManager.scala

```diff
@@ -47,17 +47,21 @@ case class StateManager(depositDir: File, submitBase: File, easyHome: URL) exten
    *          unless more recent values might be available in SUBMITTED/UUID/deposit.properties
    */
   def getStateInfo: Try[StateInfo] = Try {
-    toDraftState(getProp(stateLabelKey, draftProps)) match {
-      case State.submitted | State.inProgress =>
-        val newState: StateInfo = getProp(stateLabelKey, submittedProps) match {
-          case "SUBMITTED" => StateInfo(State.submitted, getStateDescription(draftProps))
-          case "REJECTED" => StateInfo(State.rejected, getStateDescription(submittedProps))
-          case "FAILED" => StateInfo(State.inProgress, s"The deposit is in progress.")
-          case "IN_REVIEW" => StateInfo(State.inProgress, s"The deposit is available at $landingPage")
-          case "FEDORA_ARCHIVED" | "ARCHIVED" => StateInfo(State.archived, s"The dataset is published at $landingPage")
-          case str: String =>
+    getDraftState match {
+      case draftState @ (State.submitted | State.inProgress) =>
+        val newState: StateInfo = Try(getProp(stateLabelKey, submittedProps)) match {
+          case Success("SUBMITTED") => StateInfo(State.submitted, getStateDescription(draftProps))
+          case Success("REJECTED") => StateInfo(State.rejected, getStateDescription(submittedProps))
+          case Success("FAILED") => StateInfo(State.inProgress, s"The deposit is in progress.")
+          case Success("IN_REVIEW") => StateInfo(State.inProgress, s"The deposit is available at $landingPage")
```

Comment:
Not directly related to this issue, but can you try creating an HTML string with an `<a></a>` for the `$landingPage` instead of this 'normal' string here?
In the deposit-ui, use [`dangerouslySetInnerHTML`](https://reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml), as we also did [here](https://github.com/DANS-KNAW/easy-deposit-ui/blob/0fa0e938a8d75ac70f9aaa26a8df0c341d40e3e5/src/main/typescript/lib/formComponents/HelpText.tsx#L55).

Reply:
Create an issue to implement your suggestion when we are done with the current complex conflicting pull requests on the deposit-ui.